package.json

```diff
@@ -14,6 +15,20 @@
   },
   "homepage": "https://github.com/18F/vote-gov#readme",
   "devDependencies": {
-    "gulp": "^3.9.1"
+    "browserify": "^13.0.0",
+    "del": "^2.2.0",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^2.0.0",
+    "gulp-filter": "^3.0.1",
+    "gulp-rename": "^1.2.2",
+    "gulp-sass": "^2.2.0",
+    "gulp-scss-lint": "^0.3.9",
+    "gulp-uglify": "^1.5.2",
+    "gulp-util": "^3.0.7",
+    "jquery": "^1.12.1",
+    "run-sequence": "^1.1.5",
+    "uswds": "^0.8.5",
+    "vinyl-buffer": "^1.0.0",
+    "vinyl-source-stream": "^1.1.0"
```

Comment:
@theresaanna brought up a good point on whether all these dependencies are necessary right now. 💭 I think we can safely remove them when the updated PR around splitting the gulp into config files is worked on / done.